modules/xmpp/JingleSessionPC.js

```diff
@@ -87,7 +106,8 @@ JingleSessionPC.prototype.doInitialize = function () {
     this.peerconnection = new TraceablePeerConnection(
             this.connection.jingle.ice_config,
             RTC.getPCConstraints(),
-            this);
+            this.room.options,
```

Comment:
how about we copy the options and pass them along? it looks a bit weird that we pass some ChatRoom options to a PC.